devops/.env.sample

```diff
@@ -1,11 +1,13 @@
-# Used by backend state 
-MGMT_STORAGE_ACCOUNT_NAME=__CHANGE_ME__
+ARM_TENANT_ID=__CHANGE_ME__
+ARM_SUBSCRIPTION_ID=__CHANGE_ME__
+ARM_CLIENT_ID=__CHANGE_ME__
+ARM_CLIENT_SECRET=__CHANGE_ME__
+SUB_ID=__CHANGE_ME__
```

Comment:
Would people know that SUB_ID is subscription ID?

Reply:
You caught a "bug" - that was not suppose to be there anymore. Thanks!